kubeadm/scripts/PrepareNode.ps1

```diff
@@ -89,8 +89,14 @@ start-service rancher-wins
 mkdir -force C:\var\log\kubelet
 mkdir -force C:\var\lib\kubelet\etc\kubernetes
 mkdir -force C:\etc\kubernetes\pki
-New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C:\etc\kubernetes\pki\
 
+$cmd_commands=@("C:\k\kubelet.exe ", "`$global:KubeletArgs ", "--cert-dir=`$env:SYSTEMDRIVE\var\lib\kubelet\pki ", "--config=/var/lib/kubelet/config.yaml ", "--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf ", "--kubeconfig=/etc/kubernetes/kubelet.conf ", "--hostname-override=`$(hostname) ", "--pod-infra-container-image=```"mcr.microsoft.com/oss/kubernetes/pause:3.6```" ", "--enable-debugging-handlers ", "--cgroups-per-qos=false ", "--enforce-node-allocatable=```"```" ", "--resolv-conf=```"```" ", "--log-dir=/var/log/kubelet ", "--logtostderr=false ")
+[version]$CurrentVersion = $($KubernetesVersion.Split("v") | Select -Index 1)
+[version]$V1_24_Version = '1.24'
+if ($CurrentVersion -lt $V1_24_Version) {
```

Comment:
please add a note here stating that these config options were removed in Kubernetes v1.24, and link to the release notes.